accounts/abi/bind/util_test.go

```diff
@@ -119,7 +119,7 @@ func TestWaitDeployedCornerCases(t *testing.T) {
 	}
 
 	// Create a transaction that is not mined.
-	tx = types.NewContractCreation(1, big.NewInt(0), 3000000, big.NewInt(1), common.FromHex(code))
+	tx = types.NewContractCreation(1, big.NewInt(0), 3000000, big.NewInt(1), common.FromHex(code), nil, nil)
```

Comment:
A general comment, maybe the testcases for feeCapPerGas and maxMinerBribePerGas not being nil scenarios also should be added in all related unit test.